pyramid_oereb/lib/renderer/extract/templates/xml/public_law_restriction.xml

```diff
@@ -65,11 +71,7 @@
         <AreaUnit>${public_law_restriction.area_unit}</AreaUnit>
 %endif
 %if public_law_restriction.length_share:
```

Comment:
you are testing `length_share` here. To me it makes more sense to directly test for `length_unit`

Reply:
Good catch